{"uid":"5c5c8e3c26cb46da","name":"test_config_parameters","fullName":"tests.integration.test_config#test_config_parameters","historyId":"c19e5bad9d6cbffdfc622fd5a4145171","time":{"start":1725936278102,"stop":1725937290665,"duration":1012563},"description":"Build and deploy one unit of PostgreSQL and then test config with wrong parameters.","descriptionHtml":"<p>Build and deploy one unit of PostgreSQL and then test config with wrong parameters.</p>\n","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/tests/integration/test_config.py\", line 25, in test_config_parameters\n    await build_and_deploy(ops_test, 1)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/tests/integration/helpers.py\", line 102, in build_and_deploy\n    await model.wait_for_idle(\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2970, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nasyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1725936277855,"stop":1725936277856,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1725936277856,"stop":1725936277856,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1725936277856,"stop":1725936278102,"duration":246},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1725936278102,"stop":1725936278102,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Build and deploy one unit of PostgreSQL and then test config with wrong parameters.","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/tests/integration/test_config.py\", line 25, in test_config_parameters\n    await build_and_deploy(ops_test, 1)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/tests/integration/helpers.py\", line 102, in build_and_deploy\n    await model.wait_for_idle(\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2970, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nasyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent","steps":[],"attachments":[{"uid":"58ba11e7ba41ee61","name":"log","source":"58ba11e7ba41ee61.txt","type":"text/plain","size":4909}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1725937290668,"stop":1725937290668,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1725937290669,"stop":1725937291079,"duration":410},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1725937291080,"stop":1725937291080,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1725937291080,"stop":1725937291080,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1725937291080,"stop":1725937291080,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_config"},{"name":"host","value":"github-runner"},{"name":"thread","value":"15996-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_config"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"764091311bf823dc","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":28,"skipped":0,"passed":53,"unknown":13,"total":94},"items":[{"uid":"b1aee5221048ca","reportUrl":"../170//#testresult/b1aee5221048ca","status":"passed","time":{"start":1725847285363,"stop":1725847813578,"duration":528215}},{"uid":"d549992e1bad7ca6","reportUrl":"../169//#testresult/d549992e1bad7ca6","status":"passed","time":{"start":1725761227041,"stop":1725761969532,"duration":742491}},{"uid":"3fe15cb5d794878","reportUrl":"../168//#testresult/3fe15cb5d794878","status":"passed","time":{"start":1725675457460,"stop":1725675983865,"duration":526405}},{"uid":"ed8afa7b9e9b5ad4","reportUrl":"../167//#testresult/ed8afa7b9e9b5ad4","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent","time":{"start":1725588136032,"stop":1725589153689,"duration":1017657}},{"uid":"a9f3942651b001ee","reportUrl":"../166//#testresult/a9f3942651b001ee","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent","time":{"start":1725502080796,"stop":1725503092569,"duration":1011773}},{"uid":"e6497baba1581f4b","reportUrl":"../165//#testresult/e6497baba1581f4b","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent","time":{"start":1725415572656,"stop":1725416590991,"duration":1018335}},{"uid":"11601d4b0b73e5d8","reportUrl":"../164//#testresult/11601d4b0b73e5d8","status":"passed","time":{"start":1725329471118,"stop":1725330005108,"duration":533990}},{"uid":"760a69e88f725436","reportUrl":"../163//#testresult/760a69e88f725436","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent","time":{"start":1725242446264,"stop":1725243466459,"duration":1020195}},{"uid":"c3c414aa054a69d2","reportUrl":"../162//#testresult/c3c414aa054a69d2","status":"passed","time":{"start":1725156536519,"stop":1725157243593,"duration":707074}},{"uid":"963aa1a758f42717","reportUrl":"../161//#testresult/963aa1a758f42717","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent","time":{"start":1725070727016,"stop":1725071746017,"duration":1019001}},{"uid":"cc4d84932bdcf082","reportUrl":"../160//#testresult/cc4d84932bdcf082","status":"passed","time":{"start":1724984402340,"stop":1724984696190,"duration":293850}},{"uid":"ee107df0e814b2ed","reportUrl":"../159//#testresult/ee107df0e814b2ed","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent","time":{"start":1724898618822,"stop":1724899631811,"duration":1012989}},{"uid":"ec182968847a519f","reportUrl":"../158//#testresult/ec182968847a519f","status":"passed","time":{"start":1724809147224,"stop":1724809707545,"duration":560321}},{"uid":"a168c92f96004979","reportUrl":"../157//#testresult/a168c92f96004979","status":"passed","time":{"start":1724724909543,"stop":1724725481736,"duration":572193}},{"uid":"162094c29095e2a9","reportUrl":"../156//#testresult/162094c29095e2a9","status":"unknown","time":{}},{"uid":"e11d43612d7feb1f","reportUrl":"../155//#testresult/e11d43612d7feb1f","status":"passed","time":{"start":1724551872112,"stop":1724552240411,"duration":368299}},{"uid":"ffbd4b4a0b303d4d","reportUrl":"../154//#testresult/ffbd4b4a0b303d4d","status":"passed","time":{"start":1724465401006,"stop":1724465624585,"duration":223579}},{"uid":"da5c2095fffd0d5f","reportUrl":"../153//#testresult/da5c2095fffd0d5f","status":"passed","time":{"start":1724378071344,"stop":1724378433410,"duration":362066}},{"uid":"e776b84b2ad0cfed","reportUrl":"../152//#testresult/e776b84b2ad0cfed","status":"passed","time":{"start":1724292716969,"stop":1724293426621,"duration":709652}},{"uid":"e0f97c9cd4edb34f","reportUrl":"../151//#testresult/e0f97c9cd4edb34f","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\npostgresql-k8s/0 [allocating] waiting: installing agent","time":{"start":1724206423913,"stop":1724207437547,"duration":1013634}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"5c5c8e3c26cb46da.json","parameterValues":[]}